pkg/machines/components/aggregateStatusCards.css

```diff
@@ -44,6 +44,10 @@
   }
```

Comment:
We're nesting here, as if this were SCSS, but the file has a CSS extension. Was this code moved?